docs/plugins.md

```diff
@@ -347,6 +347,25 @@ const config = {
 
 <img src="../media/plugins/hexColor.png" width="319">
 
+#### `accelerator`
+
+```js
+const config = {
+  shortcut: {
+    title: 'Trigger Shortcut',
+    customType: 'accelerator',
```

Comment:
Let's call it `keyboardShortcut`. Accelerator is not a common term for this.

Reply:
Renamed 👍